src/devices/Arduino/tests/README.md

```diff
@@ -0,0 +1,3 @@
+# How to run the Arduino Firmata tests
+
+it should be explicit use: `dotnet test --filter feature=firmata`
```

Comment:
```suggestion
In order to run all Firmata tests use: `dotnet test --filter feature=firmata`
```

Reply:
Other option is to prepend this with: `Tests using hardware are disabled by default. `